public/index.html

```diff
@@ -5,10 +5,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
 
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <meta name="msapplication-TileColor" content="#00aba9">
+    <meta name="theme-color" content="#ffffff">
 
-    <title>TB Asistente Diario</title>
+    <!-- TODO: Are we going to dynamically change this? -->
+    <title>Asistente de tratamiento</title>
```

Comment:
We've got to jump through a little hoop with our `react` setup.
Does https://jaketrent.com/post/change-page-title-in-react/ look okay to you?

Reply:
Created an issue, nice find.